examples/ruby/README.md

```diff
@@ -0,0 +1,36 @@
+# Sara Alert FHIR Client App
+
+This simple example application written in the Ruby programming language illustrates how to authenticate and use the Sara Alert FHIR API, using the M2M workflow.
```

Comment:
I would just specify the exact workflow so that it aligns with what we have in the documentation. Officially it's called the "SMART on FHIR Backend Services Workflow."

Reply:
Fixed.